src/translate/compute.rs

```diff
@@ -26,16 +26,19 @@ pub fn translate_computation(meta: &TranslateMetadata, operation: ArithOp, left:
     match meta.arith_module {
         ArithType::BcSed => {
             let (left, right) = (left.unwrap_or_default(), right.unwrap_or_default());
-            let mut math_lib_flag = true;
-            // Removes trailing zeros from the expression
-            let sed_regex = "/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//";
+            let mut scale = "";
+            let mut after_bc = "";
+            let sed_regex = "| sed \"/\\./ s/\\.\\{0,1\\}0\\{1,\\}$//\"";
             let op = match operation {
                 ArithOp::Add => "+",
                 ArithOp::Sub => "-",
                 ArithOp::Mul => "*",
-                ArithOp::Div => "/",
+                ArithOp::Div => {
+                    after_bc = sed_regex;
+                    "/"
+                },
                 ArithOp::Modulo => {
-                    math_lib_flag = false;
+                    scale = "scale=0;";
```

Comment:
This doesn't appear to work:
```
$ echo 'scale=0;1.4+2.4' | bc
3.8
```
Did I miss something?

Reply:
the unit tests for the modulo works as they are now.
I suggest to check those.

Reply:
I too couldn't wrap my head around it

Reply:
that change is for the modulo stuff, with this change the tests we have right now works.